pkg/console/subresource/consoleserver/types.go

```diff
@@ -72,6 +72,12 @@ type Customization struct {
 	CustomLogoFile       string `yaml:"customLogoFile,omitempty"`
 	// developerCatalog allows to configure the shown developer catalog categories.
 	DeveloperCatalog *DeveloperConsoleCatalogCustomization `yaml:"developerCatalog,omitempty"`
+	ProjectAccess    ProjectAccess                         `yaml:"projectAccess,omitempty"`
+}
+
+// ProjectAccess contains options for project access roles
+type ProjectAccess struct {
+	AvailableClusterRoles []string `yaml:"availableClusterRoles,omitempty"`
```

Comment:
Once the PR is merged we need to add this change to https://github.com/openshift/console/blob/master/pkg/serverconfig/types.go with the needed console changes. Not sure if there is any PR yet.

Reply:
Yes it's in this PR https://github.com/openshift/console/pull/8502
(also ready for review)